### PR TITLE
script to copy flow source files to dist

### PR DESCRIPTION
**Pull request by kevinbarabash**
This solves the problem of how to include .js.flow library definition files in published packages.  This diff just copies over the source files into the dist folder which is an awful solution, but there aren't any good alternatives since `flow gen-flow-files` doesn't work and writing library definitions from hand is really time consuming.  https://medium.com/netscape/shipping-flowtype-definitions-in-npm-packages-c987917efb65 has a good write up about this technique.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 8e4cd9ed8cc42fdfea4c99009fd9939aaeeeed66

https://deploy-preview-113--wonder-blocks.netlify.com

**Comment by kevinbarabash** [comment]
Another option is not to copy the files and instead set dist/index.js.flow to:
```
// @flow
export * from '../src';
```
and then include `src` in the files when publishing.  This approach feels functionally equivalent and means we can avoid the copy step.

File: utils/copy-flow-source.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const flowCopySource = require("flow-copy-source");
```
Comment (jeresig) [inline utils/copy-flow-source.js]: 🛑 A quick comment here explaining what this file is used for would be good!

File: utils/copy-flow-source.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const path = require("path");
+const fs = require("fs");
+
+const packages = fs.readdirSync(path.join(__dirname, "../packages"));
+
+for (const pkg of packages) {
+    // eslint-disable-next-line
```
Comment (jeresig) [inline utils/copy-flow-source.js]: 🛑 Would be good to include the rule that is being blocked, rather than blocking all errors.

**Pull request merged**